src/PDS4_NH_IngestLDD.xml

```diff
@@ -1970,16 +2184,8 @@
     </DD_Rule_Statement>
   </DD_Rule>
   
-  <!-- NEEDED: Frame_Ancillary_Data Rules -->
-  
-  <!-- NEEDED: Inst_Details class choice list -->
-  
-  <!-- + number_of_frames must match count(Frame_Associations) -->
-  <!-- + frame_number must be in range [0-(number_of_frames-1)] -->
-  <!-- + any *_local_identifier value must match a local_identifier in the label -->
-
```

Comment:
We should be sure to make a note of these somewhere else if they are still eventually desired and these comments are to be removed.

Reply:
I checked them over and most of them are no longer relevant - the references names and classes no longer exist in the dictionary, and I don't think a choice list improves the Instrument Details handling enough to justify the annoyance, although we can discuss it if you're interested...